common/company_types/00_companies_europe.txt

```diff
@@ -382,8 +382,8 @@ company_gotaverken = {
 	}
 	
 	prosperity_modifier = {
-		unit_navy_defense_mult = 0.1
-		unit_navy_offense_mult = 0.05
+		interest_group_ig_armed_forces_pop_attraction_mult = 0.05
+		interest_group_ig_industrialist_approval_add = 2
```

Comment:
Shipyard company, I also think companies should have _at least_ 1 economic bonus.

Reply:
for the prosperity modifier specifically? because they technically do for all related industries attached to them. some companies don't provide one bonus at all, instead doing things like giving influence or prestige.